build/template.html

```diff
@@ -30,5 +30,7 @@ <h1>{heading}</h1>
 		</div>
 	</div>
 	<script src="{script}"></script>
+
+	<script data-goatcounter="https://hunterosc.goatcounter.com/count" async src="//gc.zgo.at/count.js"></script>
```

Comment:
This is taken directly from https://www.goatcounter.com/help/start. There are a few things going on here...

The script `count.js` is fetched asychronously from https://gc.zgo.at/count.js (so as to not slow down the rendering of the page). It will then ping the site we set with the `data-goatcounter` attribute whenever a trackable event happens (e.g. the page is visited).

We could grab a copy of the `count.js` script and keep it in this repository instead of fetching it from their CDN, but I didn't see a good reason to. We could also look to self-host GoatCounter in the future if we wanted to, and point `data-goatcounter` at our instance, but I think it would be overkill.